src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -3611,30 +3612,30 @@ public void registerDefaultRecipes() {
 				registerFuel(new MachineFuel(3, new ItemStack(Material.POTATO)));
 				registerFuel(new MachineFuel(3, new ItemStack(Material.SUGAR_CANE)));
 				registerFuel(new MachineFuel(3, new ItemStack(Material.NETHER_WART)));
-				registerFuel(new MachineFuel(2, new ItemStack(Material.DANDELION)));
-				registerFuel(new MachineFuel(2, new ItemStack(Material.POPPY)));
 				registerFuel(new MachineFuel(2, new ItemStack(Material.RED_MUSHROOM)));
 				registerFuel(new MachineFuel(2, new ItemStack(Material.BROWN_MUSHROOM)));
 				registerFuel(new MachineFuel(2, new ItemStack(Material.VINE)));
 				registerFuel(new MachineFuel(2, new ItemStack(Material.CACTUS)));
 				registerFuel(new MachineFuel(2, new ItemStack(Material.LILY_PAD)));
 				registerFuel(new MachineFuel(8, new ItemStack(Material.CHORUS_FRUIT)));
-
+				registerFuel(new MachineFuel(1, new ItemStack(Material.BAMBOO)));
+				registerFuel(new MachineFuel(1, new ItemStack(Material.KELP)));
+				registerFuel(new MachineFuel(2, new ItemStack(Material.DRIED_KELP)));
+				registerFuel(new MachineFuel(20, new ItemStack(Material.DRIED_KELP_BLOCK)));
+				registerFuel(new MachineFuel(1, new ItemStack(Material.SEAGRASS)));
+				registerFuel(new MachineFuel(2, new ItemStack(Material.SEA_PICKLE)));
```

Comment:
This change belongs to a different PR, so it should be excluded from this one.

Reply:
My bad, though for sure I added it as a new branch will fix shortly